ehitchhiking-ui/src/app/pages/main-screen/trip-registration/trip-registration.component.sass

```diff
@@ -0,0 +1,37 @@
+@import "../../../../styles/colors"
+$trip-registration-form-height: 18rem
+$trip-registration-form-width: 20rem
+.trip-registration-form
+  position: relative
+  z-index: 2 // more than map z-index, and menu z-index
+  background-color: $white
+  height: $trip-registration-form-height
+  width: $trip-registration-form-width
+  margin-left: auto
+  margin-right: auto
+  box-shadow: 0 0 9px rgba(0, 0, 0, 0.2)
+  border-radius: 0.2rem
+  .input-from,.input-to
```

Comment:
pls combine common styles under a new classes
For example,
.form-control
  margin-top: 0.2rem

Reply:
fixed